.swiftlint.yml

```diff
@@ -41,7 +41,7 @@ line_length:
     ignores_comments: true
     ignores_urls: true
 function_body_length:
-    warning: 50
+    warning: 65
```

Comment:
Looks like this is failing for just one function. Can you look at refactoring the function instead of increasing the body length?

Reply:
This same issue was fixed in Android by refactoring the function. https://github.com/adobe/aepsdk-edge-android/blob/main/code/edge/src/main/java/com/adobe/marketing/mobile/EdgeHitProcessor.java#L100

Reply:
I will follow that and refactor.